pkg/email/templates/system_intake_submit_business_case_requester.gohtml

```diff
@@ -0,0 +1,20 @@
+{{template "easi_header.gohtml"}}
+
+<p>You have {{if .IsResubmitted}}made changes to{{else}}completed{{end}} a {{if .IsDraft}}draft{{else}}final{{end}} Business Case for your IT Governance request {{.RequestName}}. The Governance Review Team (GRT) will review {{if .IsResubmitted}}your changes{{else}}it{{end}} and {{if .IsDraft}}will either get back to you within two business days or share feedback at your scheduled GRT meeting.{{else}}get back to you within two business days. In the meantime, you may review guidance in EASi about <a href="{{.PreparingForGRBLink}}">preparing for the GRB</a>{{end}}.</p>
```

Comment:
FIGMA shows the request name in parenthesis () https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2202-14705&mode=design&t=jJeHsDYUxs9LDMoz-0

Reply:
extra punctuation after GRT meeting for resubmit. Also , it should say The Governance Review Team (GRT) instead of just GRT

Reply:
Final buisness case is missing language encouraging requester to fill out the slide deck for the GRB meeting https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2202-128037&mode=design&t=1W2JkzMlxIJZZAaQ-0

Reply:
The slide deck part was nixed because we couldn't get the link to the slide deck.